src/redux/configureStore.js

```diff
@@ -73,7 +73,15 @@ const configureStore = () => {
     const WHITE_LIST = []
     // The following actions will be ignored if not whitelisted but presents in DOMAINS
     // So, we can enable whole domain, but still exclude aome actions from domain
-    const IGNORED_ACTIONS = []
+    const IGNORED_ACTIONS = [
+      'market/ADD_TOKEN',
```

Comment:
don't push this.

Reply:
We need to push all available domains and actions. But by default all of them must be commented out. If developer need, he just uncomment it instead of copy-pasting from console.

Reply:
I suggest:
For default: show nothing
If you want, add needed logs for yourself.

Reply:
Done